src/test/resources/network/brightspots/rcv/test_data/no_one_meets_minimum/no_one_meets_minimum_config.json

```diff
@@ -1,15 +1,16 @@
 {
-  "tabulatorVersion" : "1.2.0",
-  "outputSettings" : {
-    "contestName" : "Test Set No One Meets Minimum",
-    "outputDirectory" : "output",
-    "contestDate" : "2022-08-15",
-    "contestJurisdiction" : "",
-    "contestOffice" : "",
-    "tabulateByPrecinct" : false,
-    "generateCdfJson" : false
+  "tabulatorVersion": "TEST",
```

Comment:
Oops. Btw, in this PR IntelliJ is doing that annoying thing where it reformats our JSON slightly differently from the way that our app generates it. I guess it doesn't break anything when it's just a config file (as opposed to an expected summary JSON file), but if anyone modifies this file using the app in the future, every line will change again.

Reply:
That's a pain... just submitted another commit to correct it. We should file an issue to figure out how to stop fighting this...

Reply:
Yeah. I spent a fair amount of time a couple years ago trying to figure out how to change either one of them to match the other, but never succeeded. I prefer the IntelliJ version.